README.md

```diff
@@ -40,6 +49,45 @@ In this example:
 - `password` and `other_sensitive_key` will have values replaced with `"[FILTERED]"`
 - all string values will be truncated to 100 characters
 
+## Dynamic configuration
+```elixir
+iex> require Logger
+iex> Logger.remove_backend(:console)
+iex> Logger.add_backend(StathamLogger)
+iex> Logger.configure_backend(StathamLogger, metadata: :all)
+```
+
+## Usage
+```elixir
+iex> Logger.metadata(metadata_field_1: "metadata_value_1")
+iex> Logger.debug("hello")
+
+# Output
+iex> {"file":"/some_file.ex","function":"say_hello/0","line":67,"logger":{"thread_name":"#PID<0.222.0>","method_name":"HelloModule.say_hello/0"},"message":"hello","metadata_field_1":"metadata_value_1","mfa":["HelloModule","say_hello",0],"module":"HelloModule","pid":"#PID<0.222.0>","syslog":{"hostname":"mb","severity":"debug","timestamp":"2021-10-07T10:20:17.902Z"}}
+```
+
+## Overwrite log level per invocation using options_groups
+
+1. Configure options groups
+```elixir
+config :logger, level: :info
+
+config :logger, StathamLogger,
+  options_groups: %{
```

Comment:
@gorkunov @YuriiZdyrko can we simply use `groups` instead of `options_groups`?

Reply:
Maybe `modes`?